src/index.html

```diff
@@ -4,8 +4,29 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <script src="index.js" defer></script>
 </head>
 <body>
-  <h1>Webpack</h1>
+  <h1>Leaderboard</h1>
+  <div class="container">
+    <div class="score-list-area">
+      <div class="recent-scores">
+        <h3>Recent scores</h3>
+        <button id="refresh" class="refresh">refresh</button>
+        </div>
+      <div class="scores">
+        <ul id="ul" class="added-scores">
+        </ul>
+    </div>
+    </div>
+    <div class="score-area">
+      <h3>Add your score</h3>
+      <div class="form-area">
+        <input id="name" class="name" type="text" placeholder="Enter your name">
+        <input id="score" class="score" type="text" placeholder="Enter your score">
```

Comment:
[OPTIONAL] Please, consider making this input of type `number` instead of text to avoid text inputs